Core/clim-core/table-formatting.lisp

```diff
@@ -259,33 +312,18 @@ skips intervening non-table output record structures."))
                 (replay-output-record record stream region x-offset y-offset))
             (nreverse other-records)))))
 
-(defmacro formatting-table ((&optional (stream t)
-                                       &rest args
-                                       &key x-spacing y-spacing
-                                       multiple-columns
-                                       multiple-columns-x-spacing
-                                       equalize-column-widths (move-cursor t)
-                                       (record-type ''standard-table-output-record)
-                                       &allow-other-keys)
-                            &body body)
-  (declare (ignore x-spacing y-spacing multiple-columns
-                   multiple-columns-x-spacing
-                   equalize-column-widths move-cursor record-type))
-  (gen-invoke-trampoline 'invoke-formatting-table
-                         (list (stream-designator-symbol stream
-                                                         '*standard-output*))
-                         args
-                         body))
+(eval-when (:compile-toplevel :load-toplevel :execute)
+  (alexandria:define-constant +standard-table-output-record-initargs+
+      '(x-spacing y-spacing multiple-columns
+        multiple-columns-x-spacing equalize-column-widths)
+    :test #'equal))
```

Comment:
these are initargs for table-output-record, so rename the constant.

n.b multiple-columns is not mentioned in the spec, howerver it seems to be used and is included in the Franz CLIM guide:

> multiple-columns Either nil,t, or an integer. If it istor an integer, the table rows are broken up into multiple columns. If it ist, CLIM will determine the optimal number of columns. If it is an integer, it will be interpreted as the desired number of columns.

since you are at it, please add taht entry to the spec as table-output-record initarg. use keywords, not internal symbols (ditto).

Reply:
The keyword is mentioned in the description of `formatting-table` (with a similar description to the one you quote) so it seems just the initarg is missing.